src/linagora.esn.contact/app/services/contact-restangular.service.js

```diff
@@ -6,7 +6,7 @@ const Restangular = require('restangular');
   angular.module('linagora.esn.contact')
     .factory('contactRestangularService', contactRestangularService);
 
-  function contactRestangularService(httpErrorHandler) {
+  function contactRestangularService(Restangular, httpErrorHandler) {
```

Comment:
What is this dark sorcery? How the hell did this even work before? :trollface: (this is not a part of the review though)

Reply:
It works before because when we did the task explicit dependencies, someone removed this variable